### PR TITLE
Update vite.mdx for pnpm & yarn sidecar

### PR DESCRIPTION
**Pull request by alesvaupotic**
Before opening this PR:
* [ ] I added a [Changeset Entry](https://spotlightjs.com/contribute/changesets/) with `pnpm changeset:add`
* [ ] I referenced issues that this PR addresses

Just updated the docs.

### CODE REVIEW

File: packages/website/src/content/docs/sidecar/vite.mdx [hunk 1]
```diff
@@ -14,12 +14,12 @@ We provide a Vite plugin that automatically starts & stops Sidecar for you when
   </TabItem>
   <TabItem label="pnpm">
     ```sh
-    pnpm add @spotlightjs/spotlight --save-dev
```
Comment (Shubhdeep12) [inline packages/website/src/content/docs/sidecar/vite.mdx:17]: Hi @alesvaupotic this command is correct. 
Other places are affected I guess.

it should @spotlightjs/spotlight

cc @Lms24

Comment (Lms24) [reply packages/website/src/content/docs/sidecar/vite.mdx:17]: Hi, @alesvaupotic and @Shubhdeep12 good catches, thanks! I agree, this is a bug in the `spotlightjs/spotlight` package. it should also export the vite plugin. Will fix this asap.

Comment (alesvaupotic) [reply packages/website/src/content/docs/sidecar/vite.mdx:17]: Thanks, I was wondering what I was doing wrong. Trying to work it out for SvelteKit app ...

**Comment by dcramer** [comment]
Do we think this is related to https://news.ycombinator.com/item?id=38550812 ?

File: packages/website/src/content/docs/sidecar/vite.mdx (hunk 1 above)
Comment (Lms24) [reply packages/website/src/content/docs/sidecar/vite.mdx:17]: TLDR: I'd recommend setting up the sidecar within the server hooks. Easy to do and you don't need to mess around with vite plugins or npx commands.

**Comment by alesvaupotic** [comment]
Abandoning this PR as it solves nothing.

**Pull request closed without merging**